api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -93,10 +93,6 @@ public class UserServiceAccessTest {
   private static DbAccessTier registeredTier;
   private static DbAccessTier controlledTier;
 
-  private Function<Timestamp, Function<DbUser, DbUser>> registerUserWithTime =
```

Comment:
`registerUser()` did not actually use the timestamp passed to it.  Removing that parameter led to this further simplification: neither of these functions are actually needed.